packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -399,9 +416,10 @@ export const ReleaseDetailsLayout = ({
                       {formatMessage(
                         {
                           id: 'content-releases.header.actions.created.description',
-                          defaultMessage: ' by {createdBy}',
+                          defaultMessage:
+                            '{hasCreatedByUser, select, true { by {createdBy}} other { by deleted user}}',
                         },
-                        { createdBy }
+                        { createdBy: getCreatedByUser(), hasCreatedByUser }
```

Comment:
Same here, Do we really need this boolean? Can we just check if the returning value from getCreatedByUser is not a null value?

Reply:
I believe the ternary (select) inside format message is expecting a boolean. I don't think it will evaluate a value as truthy or falsy.